### PR TITLE
fix: ansible play that installed mongodb has issues and is old/not su…

### PR DESCRIPTION
**Pull request by gabeobrien**
…pported; mongodb is now installed with package managers and configured with ansible community module and regex

### CODE REVIEW

File: deploy/ansible/mongodb.yml [hunk 1]
```diff
@@ -5,17 +5,80 @@
...
-    mongodb_users:
-      - {
-          name: "{{ mongodb_user }}",
-          password: "{{ mongodb_pass }}",
-          roles: readWrite,
-          database: "{{ mongodb_db }}"
-        }
+  become: true
+  tasks:
+    - name: Install aptitude using apt
```
Comment (ipcrm) [inline deploy/ansible/mongodb.yml:10]: do we need this?

File: deploy/ansible/mongodb.yml [hunk 2]
```diff
@@ -5,17 +5,80 @@
...
+      apt:
+        name: aptitude
+        state: latest
+        update_cache: yes
```
Comment (ipcrm) [inline deploy/ansible/mongodb.yml:14]: Do we need this?

**Review by ipcrm — CHANGES_REQUESTED** [review]
Looks great! One less dependency! 

Had a single question on this.

File: deploy/ansible/mongodb.yml (hunk 2 above)
Comment (gabeobrien) [reply deploy/ansible/mongodb.yml:14]: Good call.  I was meant to test without is. I do not think we need it. Will confirm.

Comment (gabeobrien) [reply deploy/ansible/mongodb.yml]: Removed!

**Review by ipcrm — APPROVED** [review]

**Pull request merged**